programs/voter-stake-registry/src/instructions/configure_voting_mint.rs

```diff
@@ -31,13 +31,23 @@ pub struct ConfigureVotingMint<'info> {
 /// This instruction can be called several times for the same mint and index to
 /// change the voting mint configuration.
 ///
-/// The vote weight for `amount` of native tokens will be
+/// The vote weight for `amount` of native tokens will be one of these
+/// 
+/// With min_lockup_saturation_secs == 0
 /// ```
 /// vote_weight =
 ///     amount * 10^(digit_shift)
 ///            * (baseline_vote_weight_scaled_factor/1e9
 ///               + lockup_duration_factor * max_extra_lockup_vote_weight_scaled_factor/1e9)
 /// ```
+/// 
+/// With min_lockup_saturation_secs > 0
```

Comment:
Is this comment accurate anymore? I don't see this math.

Reply:
No, should be `/// With min_lockup_saturation_secs > 0 && lockup.total_seconds == min_lockup_saturation_secs`